util/fixtures/makeWorkorder.js

```diff
@@ -15,6 +15,7 @@ module.exports = function makeWorkorder(userId, workflowId) {
     finishDate: "2029-02-23T00:00:00Z",
     finishTime: "2029-02-23T00:00:00Z",
     summary: "Do the work",
+    // TODO: for some reason 'startTimestamp' now shows up as 'finish date/time' in the portal
```

Comment:
@paolobueno also, given that it's `startTimestamp` that seems to populate those fields now, should it be set to some time in the future, like `finishDate` and `finishTime` were above?